### PR TITLE
adding centralised log management

### PR DESCRIPTION
**Pull request by paulmech**
this is rough as guts - please go hard

### CODE REVIEW

File: _posts/2020-09-10-centralised-log-management.markdown [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+* listing bucket files
+* bucket file fetching
+* content decryption
+* tcp negotiation
+* ssl handshaking
+
+For a few files the effect is negligible, however when you scale it to thousands of files and many concurrent users, it is very noticeable. The solution is to periodically create *rollups* <sup>[2]</sup> of those files. This is commonly done with Spark jobs that will aim to
+
+* rewrite data into columnar formats such as Parquet for optimized reading
```
Comment (Hamo69) [inline _posts/2020-09-10-centralised-log-management.markdown:121]: optimized *should be* optimised?

File: _posts/2020-09-10-centralised-log-management.markdown [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+There are many different ways to achieve this goal - a few are outlined below
+
+|method|pros|cons|
+|-|-|-|
+|EMR Cluster Spark Jobs| - pre-baked hadoop environment<br/>- transient clusters with spot instances could be cheaper|- high complexity|
+|Glue PySpark/Scala Jobs| - serverless<br/>- simple | - More expensive at scale|
+|Airflow / Luigi, etc.| - Full Workflow Management for your ETL | - not cloud native<br/>- operational complexity|
+|Kinesis Firehose Transformations| - Embedded transformation<br/>- Backup mode to allow capturing raw data|- Poor Documentation<br/>- May see poor economies of scale with high throughput|
+
```
Comment (Hamo69) [inline _posts/2020-09-10-centralised-log-management.markdown:135]: Does it need some kind of wrap / close out?

**Review by Hamo69 — APPROVED** [review]

File: _posts/2020-09-10-centralised-log-management.markdown (hunk 2 above)
Comment (paulmech) [reply _posts/2020-09-10-centralised-log-management.markdown:135]: yes

File: _posts/2020-09-10-centralised-log-management.markdown (hunk 1 above)
Comment (paulmech) [reply _posts/2020-09-10-centralised-log-management.markdown:121]: yes

**Comment by MechRock-Marcelo** [comment]
1.. On Premise? or On-Premise:  for an organisation generating **log data on premise**? = should on premise be on-premise?  

2.. Grammer: Kinesis Firehose and its compatible agents to effectively **replace the both log shippers** and the Logstash component

Great read..

**Review by MechRock-Marcelo — APPROVED** [review]

**Pull request merged**